src/js/main.js

```diff
@@ -127,7 +127,8 @@ var main = main || {};
             // attach the generated options to the dom
             ns.selectOptionEl.html(selectOptions);
         }).fail(function(err) {
-            // TODO: create error handler
+            $('.toast').attr('style','');
```

Comment:
Is this necessary?

Reply:
That is to get rid of the style element "display:none" when toast is triggered. The reason I put "display:none" is that there was some empty space where the div is located.